creator-node/src/app.ts

```diff
@@ -1,16 +1,16 @@
-const express = require('express')
-const bodyParser = require('body-parser')
-const cors = require('cors')
-const prometheusMiddleware = require('express-prom-bundle')
-const _ = require('lodash')
-
-const DiskManager = require('./diskManager')
-const { sendResponse, errorResponseServerError } = require('./apiHelpers')
-const { logger, loggingMiddleware } = require('./logging')
-const {
-  readOnlyMiddleware
-} = require('./middlewares/readOnly/readOnlyMiddleware')
-const {
+import type { Request, Response, NextFunction, IRoute } from 'express'
+import type Logger from 'bunyan'
```

Comment:
nit: I've been adding a space after `import type` blocks so we should do that here or remove the space elsewhere. not a big deal